containers/Connector/Connector.tsx

```diff
@@ -43,12 +43,27 @@ const useConnector = () => {
 			// TODO: need to verify we support the network
 			const networkId = await getDefaultNetworkId();
 
-			// @ts-ignore
-			const provider = new ethers.providers.InfuraProvider(
-				networkId,
-				process.env.NEXT_PUBLIC_INFURA_PROJECT_ID
-			);
-
+			let provider
```

Comment:
I could have sworn our linter and precommit hooks enforced semi-colons 🤔

Reply:
Haha that's weird. I did just check and no, I didn't ignore hooks with `git commit --no-verify`.

Reply:
Take another look @JChiaramonte7

Reply:
thank you ser